General-Features.md

```diff
@@ -19,7 +19,13 @@ It's possible to hide the sidebar by pressing the left arrow next to the minimiz
 
 ### Profiles
 
-As of the 1.9.11 release, profiles are now available to hold different distinct settings configurations within the same client instance.  By default the profile loaded on startup will be the one that was most recently active, to override this you can pass `--profile=<profilename>` to the launcher and it will load that specific profile on startup.
+Profiles are separate sets of plugins and settings that can be switched between at any time. They can be managed under a tab in configuration panel.
+
+![](img/profiles.png)
+
+Below each profile are buttons to interact with them. From left to right, the buttons are: rename, duplicate, export to file, toggle cloud sync, and delete. Cloud sync is available when you are logged in to the client using a runelite.net account. Profiles can be dragged to re-order them.
```

Comment:
There's also the link button to link the config profile to an RSProfile, currently only when logged in to the game.

Additionally, consider letting people know how cloud sync enabled looks like (orange = enabled).

Reply:
I wasn't aware of the ability to link to an RSProfile. I'll look into that and mention it

I suppose that mentioning orange means enabled can't hurt either.

Reply:
Added a mention to the "link to runescape account" button, another mention for orange means enabled, and updated the inlined image to show all the buttons